saleor/graphql/channel/mutations/channel_update.py

```diff
@@ -51,10 +51,16 @@ class Arguments:
         )
 
     class Meta:
-        description = "Update a channel."
+        description = (
+            "Update a channel.\n\n"
+            "Requires one of the following permissions: MANAGE_CHANNELS.\n"
```

Comment:
@maarcingebala Any suggestion to make the description more readable?

Reply:
I think it's ok.